### ISSUE TITLE
Comparison with DragNUWA

### ISSUE THREAD

**Issue by advenTure423** [showlab/DragAnything#5]
Hi, good work. However, DragNUWA supports multi-point control. I suppose it's better to compare your work with that version.

**Comment by zhw-zhang** [comment]
yes, I also want to see this comparison results, sometimes, dragnvwa can perform pretty well if you give it some suitable trajectory lines.

**Comment by weijiawu** [comment]
Thank you for your attention. We have released the relevant inference code. Feel free to compare it at your convenience.

**Comment by zhw-zhang** [comment]
> Thank you for your attention. We have released the relevant inference code. Feel free to compare it at your convenience.

Thanks for the reply, and I also want to remind you that the trainable and freeze icons in Figure 4 are reversed. I look forward to your open source of the complete code soon.
